src/ts/components/timelinemarkers.ts

```diff
@@ -0,0 +1,208 @@
+import { PlayerAPI } from 'bitmovin-player';
+import { UIInstanceManager } from '../uimanager';
+import { DOM } from '../dom';
+import { Component, ComponentConfig } from './component';
+import { TimelineMarker } from '../uiconfig';
+import { SeekBarMarker } from './seekbar';
+import { PlayerUtils } from '../playerutils';
+import { Timeout } from '../timeout';
+
+export interface MarkersConfig extends ComponentConfig {
+  /**
+   * Used for seekBar marker snapping range percentage
+   */
+  snappingRange?: number;
+}
+
+export class TimelineMarkers extends Component<MarkersConfig> {
```

Comment:
I'm not fully happy with the name of this class.

So this is now a new `component` holding all markers right? Although it does not render itself? The seekbar is still handling / rendering the markers right?

That indicates to me that this is not a `Component`.

This looks like a `TimelineMarkersHelper` / `TimelineMarkersHandler` or something like that.

It also doesn't need to extend the `Component` as it's not added to the `Components` tree. That's not how this is designed to work.

Reply:
Well, `seekbar` is holding container, but `TimelineMarkers` are the ones responsible for filling in that container. 
I was not sure of my self. I wanted `TimelineMarkers` to be handling all of this, but it was not possible :/ 
Will create it as a helper and define `prefixCss` inside, to cover that as well.

Reply:
Done in https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/a5e96ef084255a89445df0900476b6978394f847 and https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/dd1b366fa704601e44eaa0964bb8871a34d0b6af